src/main/java/org/jitsi/videobridge/OctoChannel.java

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.colibri.*;
+import org.jitsi.service.neomedia.*;
+import org.jitsi.util.*;
+import org.jitsi.videobridge.octo.*;
+
+import java.net.*;
+import java.util.*;
+
+/**
+ * Represents an Octo channel, used for bridge-to-bridge communication.
+ *
+ * @author Boris Grozev
+ */
+public class OctoChannel
+    extends RtpChannel
+{
+    /**
+     * The {@link Logger} used by the {@link RtpChannel} class to print debug
+     * information. Note that instances should use {@link #logger} instead.
+     */
+    private static final Logger classLogger
+        = Logger.getLogger(OctoChannel.class);
+
+    /**
+     * The Octo ID of the conference, configured in {@link Conference} as the
+     * global ID ({@link Conference#getGid()}).
+     */
+    private final String conferenceId;
+
+    /**
+     * The {@link MediaType} for this channel.
+     */
+    private final MediaType mediaType;
+
+    /**
+     * The {@link TransportManager} if this channel.
+     */
+    private OctoTransportManager transportManager;
+
+    /**
+     * The {@link org.ice4j.socket.DatagramPacketFilter} which (only) accepts
+     * RTP packets for this channel.
+     */
+    private RtpChannelDatagramFilter rtpFilter
+        = new OctoDatagramPacketFilter(false);
```

Comment:
I'm wishing we had sub-types `RtpChannelRtpFilter` and `RtcpChannelRtcpFilter` instead of having to pass true/false into these all the time (even if the sub-types did nothing but pass the true/false to the parent type)